src/main/resources/com/xebialabs/deployit/ci/workflow/XLDeployDeployStep/help-environmentId.html

```diff
@@ -0,0 +1,4 @@
+<div>
+    Id of the environment in xl-deploy on which artifact needs to be deployed. Jenkin's environment variables can be used to specify this value.
```

Comment:
ID of the environment in XL Deploy to which the artifact should be deployed. Jenkin's environment variables can be used to specify this value.